pkg/internalapis/featurestates/featurestates.go

```diff
@@ -263,7 +267,8 @@ func configMapAdded(obj interface{}) {
 			featureState.Name = feature
 			featureState.Enabled, err = strconv.ParseBool(state)
 			if err != nil {
-				log.Warnf("failed to parse feature state value: %v for feature: %q", state, feature)
+				log.Errorf("failed to parse feature state value: %v for feature: %q", state, feature)
+				return
```

Comment:
Since this is a issue that can occur at dev time, can we invoke os.Exit to crash the container?

Reply:
On more thinking, I think we should crash CSI once the FSS has invalid entries or the FSS itself is deleted. Silently ignoring the problem will be problematic.

Reply:
crashing the container is a good idea. so during dev and fvt testing, we can identify the issue earlier.
making the following change.

```
                       featureState.Enabled, err = strconv.ParseBool(state)
			if err != nil {
				log.Errorf("failed to parse feature state value: %v for feature: %q", state, feature)
				os.Exit(1)
			}
```